src/ifElse.test.js

```diff
@@ -1,11 +1,41 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
```

Comment:
revise 'Checks' tab, npm run test return error, revise your code and make sure that all checks passed and resend task on review

Reply:
All my tests passed. 
![Tests run](https://github.com/mate-academy/qa_if-else-function/assets/48523402/9625838f-fb82-4aaa-964d-f32b4f48a870)
I can't resolve this error because it has no relation to my test file. 
![npm test error](https://github.com/mate-academy/qa_if-else-function/assets/48523402/43eb1525-e1a4-4d1c-a1fe-600916725c90)